jmx/plugins/org.jboss.tools.jmx.jvmmonitor.tools/src/org/jboss/tools/jmx/jvmmonitor/internal/tools/JvmAttachHandler.java

```diff
@@ -233,20 +234,22 @@ public String getLocalConnectorAddressInternal(Object monitoredVm, int pid) thro
         try {
             virtualMachine = ToolsCore.attach(pid);
 
-            String javaHome = ToolsCore.getSystemProperties(virtualMachine)
-                    .getProperty(IConstants.JAVA_HOME_PROPERTY_KEY);
+            if(!JavaUtilities.isJigsawRunning()) {

```

Comment:
please note that it would be more accurate if we are checking if jigsaw is running on the targeted vm

Reply:
I created another set of PR to handle it more accurately: https://github.com/jbosstools/jbosstools-base/pull/642 and https://github.com/jbosstools/jbosstools-server/pull/620